src/constants/timezone.constants.ts

```diff
@@ -0,0 +1,3 @@
+import { ITimezoneOption } from "@interfaces/components";
+
+export const defaultTimezone: ITimezoneOption = { label: "UTC", value: "UTC" };
```

Comment:
Two different default values exist:
  - defaultTimezone = { label: "UTC", value: "UTC" } (timezone.constants.ts)
  - defaultTimezoneValue = "Etc/GMT" (global.constants.ts)

These are different timezone identifiers. "UTC" and "Etc/GMT" are technically equivalent but having both could cause confusion.

Can we reuse them somehow?